lib/uart_blob/Kconfig

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "UART BLOB transfer"
+	depends on UART_COBS
+
+config UART_BLOB
+	bool
+	help
+	  Enabled whenever either the UART_BLOB_TX or UART_BLOB_RX modules are
+	  enabled.
+
+# Default config values 
+
+if UART_BLOB
+
+config UART_BLOB_DEFAULT_SEND_TIMEOUT
+	int
```

Comment:
If this does not have a prompt then it is not possible to set its value through prj.conf files or menuconfig. The only way this is configurable at the moment is through kconfig logic in kconfig source files.

Reply:
I intended for those default values to be invisible and just wanted a common place to store the default values for UART_BLOB_RX_SEND_TIMEOUT/UART_BLOB_TX_SEND_TIMEOUT. I can inline the value to avoid confusion.